docs/contributing/blog-and-website-contributions.md

```diff
@@ -11,50 +11,36 @@ Here are some things to keep in mind when deciding where to contribute to Gatsby
-
-To add a new blog post to the gatsbyjs.org blog:
-
-- Clone [the Gatsby repo](https://github.com/gatsbyjs/gatsby/) and navigate to `/www`.
-- Run `yarn` to install all of the website's dependencies. ([Why Yarn?](/contributing/setting-up-your-local-dev-environment#using-yarn))
-- Run `npm run develop` to preview the blog at `http://localhost:8000/blog`.
-- The content for the blog lives in the `/docs/blog` folder. Make additions or modifications here.
-- Add your avatar image to `/docs/blog/avatars`.
-- Add your name to `/docs/blog/author.yaml`.
-- Add a new folder following the pattern `/docs/blog/yyyy-mm-dd-title`. Within this newly created folder, add an `index.md` file.
-- Add `title`, `date`, `author`, `excerpt`, and `tags` to the frontmatter of your `index.md`. You can [view existing tags](/blog/tags/), or [add a new one](https://github.com/gatsbyjs/gatsby/blob/master/www/src/data/tags-docs.js) if you feel your tag merits being its own tag, though we encourage you to use existing tags.
-- If you are cross posting your post, you can add `canonicalLink` for SEO benefits. You can check the other blog posts in `/docs/blog` for examples.
-- If your blog post contains images, add them to your blog post folder and reference them in your post's `index.md`.
-- Ensure any links to gatsbyjs.org are relative links - `/contributing/how-to-contribute/` instead of `https://gatsbyjs.org/contributing/how-to-contribute`.
-- Follow the [Style Guide](/contributing/gatsby-style-guide/#word-choice) to make sure you're using the appropriate wording.
-- Double check your grammar and capitalize correctly.
-- Commit and push to your fork.
-- Create a pull request from your branch.
-  - We recommend using a prefix of `docs`, e.g. `docs/your-change` or `docs-your-change`. ([PR example](https://github.com/gatsbyjs/gatsby/commit/9c21394add7906974dcfd22ad5dc1351a99d7ceb#diff-bf544fce773d8a5381f64c37d48d9f12))
-
-### Blog post format
-
-The following format can help you in creating your new blog content. At the top is "frontmatter": a fancy name for metadata in Markdown. The frontmatter for your post should include a title, date, singular author name (for now, we would welcome issues/PRs for this), and one or more tags. Your content will follow after the second set of dashes (`---`).
-
-```md
----
-title: "Your Great Blog Post"
-date: YYYY-MM-DD
-author: Jamie Doe
-excerpt: "Here is a helpful excerpt or brief description of this blog post."
-tags:
-  - awesome
-  - post
----
-
-Your next great blog post awaits!
-
-Include images by creating a folder for your post and including
-Markdown and image files for easy linking.
-
-![awesome example](./image.jpg)
-```
+If you'd like to contribute a post to the Gatsby blog, please review the process and guidelines outlined below and submit your
+idea for the post to our [Gatsby blog proposal form](https://airtable.com/shr3449954866i3iF)
```

Comment:
i miss in the airtable form also the github user name

Reply:
With the new process, we shouldn't need their GitHub user name. They don't have to use GitHub at all if they don't want to. When it does come time to submit the blog as a PR, if the author wants to submit it themselves, we'll collect their username. (If they don't want to write their own PR, Hashim or I will do it.)